NewRelic.Platform.Sdk.FunctionalTests/RunnerTest.cs

```diff
@@ -53,5 +54,16 @@ public void TestRunnerReportContinuesWithNegativeValue()
             runner.Add(new TestAgent("FunctionalTest", -10));
             runner.SetupAndRunWithLimit(1); // Should not raise an exception
         }
+        [TestMethod]
```

Comment:
We should probably make this an ignored test by default since a 10 minute test is pretty hefty.